packages/logger/src/Logger.ts

```diff
@@ -174,23 +174,26 @@ class Logger extends Utility implements LoggerInterface {
    */
   protected isBufferEnabled = false;
 
+  /**
+   * Whether the buffer should be flushed when an error is logged
+   */
+  protected flushOnErrorLog = true;
```

Comment:
I don't think we have made this configurable, the constructor type has it but we are never updating it even when passed.

Reply:
Whoops. Sorry